app/controllers/guides_controller.rb

```diff
@@ -33,6 +34,7 @@ def update
 private
 
   def guide_params
+    params[:guide][:slug] = File.join("/service-manual/", params[:guide][:slug])
     params.require(:guide).permit(
       :slug,
       latest_edition_attributes: [
```

Comment:
It's not great what we have in this file I think. Alternatives:

## Validating input

As suggested by Andrew in an offline discussion, we could simply add a validation for the slug format and leave it to the content designer to edit it. Not the greatest UI but a very simplistic solution.

## Not storing the prefix in the local DB

Prepend `/service-manual/` before calling publishing-api. This is not great as a developer joining the project would have to be aware of this quirk, otherwise it could be very confusing.

## Encapsulating this login 

Encapsulate this logic within a model or some sort of presenter object that prepends `/service-manual` on `Edition#slug=` and removes it when displaying in the form. Could end up being a lot of extra code.

@KushalP any other ideas?